packages/@aws-cdk/aws-scheduler-alpha/test/schedule.test.ts

```diff
@@ -128,4 +128,71 @@ describe('Schedule', () => {
       },
     });
   });
-});
\ No newline at end of file
+
+  describe('flexibleTimeWindow', () => {
+    test('flexibleTimeWindow mode is set to OFF by default', () => {
+      // WHEN
+      new Schedule(stack, 'TestSchedule', {
+        schedule: expr,
+        target: new SomeLambdaTarget(func, role),
+      });
+
+      // THEN
+      Template.fromStack(stack).hasResourceProperties('AWS::Scheduler::Schedule', {
+        FlexibleTimeWindow: {
+          Mode: 'OFF',
+        },
+      });
+    });
+
+    test('flexibleTimeWindow mode can be set to FLEXIBLE', () => {
+      // WHEN
+      new Schedule(stack, 'TestSchedule', {
+        schedule: expr,
+        target: new SomeLambdaTarget(func, role),
+        flexibleTimeWindowMode: FlexibleTimeWindowMode.FLEXIBLE,
+        maximumWindowInMinutes: Duration.minutes(1440),
+      });
+
+      // THEN
+      Template.fromStack(stack).hasResourceProperties('AWS::Scheduler::Schedule', {
+        FlexibleTimeWindow: {
+          Mode: 'FLEXIBLE',
+          MaximumWindowInMinutes: 1440,
+        },
+      });
+    });
+
+    test('throw error when FlexibleTimeWindowMode is FLEXIBLE and maximumWindowInMinutes not provided', () => {
+      expect(() => {
+        new Schedule(stack, 'TestSchedule', {
+          schedule: expr,
+          target: new SomeLambdaTarget(func, role),
+          flexibleTimeWindowMode: FlexibleTimeWindowMode.FLEXIBLE,
+        });
+      }).toThrow('maximumWindowInMinutes must be provided when flexibleTimeWindowMode is set to FLEXIBLE');
+    });
+
+    test('throw error when maximumWindowInMinutes is more than 1440', () => {
```

Comment:
It is not big deal but...

```suggestion
    test('throw error when maximumWindowInMinutes is greater than 1440', () => {
```